File: frontend/src/app/movie/model/movie-list-item.ts
import { Genre } from './genre';

export class MovieListItem {
    id: number
    title: string
    releaseDate: Date
    posterPath: string
    backdropPath: string
    genres: string

    constructor(id: number, title: string, releaseDate: Date, posterPath: string, backdropPath: string, genres: string) {
        this.id = id
        this.title = title
        this.releaseDate = releaseDate
        this.posterPath = posterPath
        this.backdropPath = backdropPath
        this.genres = genres
    }
}

File: backend/MovieDatabase.API/ViewModel/Movie/MovieListItem.cs
using System;

namespace MovieDatabase.API.ViewModel.Movie
{
    public class MovieListItem
    {
        public int Id { get; set; }

        public string Title { get; set; }

        public DateTime ReleaseDate { get; set; }

        public string PosterPath { get; set; }

        public string BackdropPath { get; set; }

        public string Genres { get; set; }
    }
}
